content/doc/book/system-administration/FIPS-140.adoc

```diff
@@ -0,0 +1,54 @@
+---
+layout: section
+---
+ifdef::backend-html5[]
+ifndef::env-github[:imagesdir: ../../resources/managing]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+endif::[]
+
+= FIPS-140
+
+[IMPORTANT]
+----
+support blurb
```

Comment:
need to state that this is for informational purposes, the Jenkins project does not claim that Community does actively check for FIPS compliance in Jenkins or its plugins.